covid19-etl/etl/npi_pro.py

```diff
@@ -0,0 +1,142 @@
+import os
+import tempfile
+
+import fiona
+import geopandas as gpd
+import requests
+
+from etl import base
+from helper.format_helper import (
+    derived_submitter_id,
+    format_submitter_id,
+)
+from helper.metadata_helper import MetadataHelper
+
+CURRENT_DIR = os.path.dirname(os.path.realpath(__file__))
+
+
+class NPI_PRO(base.BaseETL):
+    def __init__(self, base_url, access_token, s3_bucket):
+        super().__init__(base_url, access_token, s3_bucket)
+
+        self.program_name = "open"
+        self.project_code = "NPI-PRO"
+        self.metadata_helper = MetadataHelper(
+            base_url=self.base_url,
+            program_name=self.program_name,
+            project_code=self.project_code,
+            access_token=access_token,
+        )
+
+        self.country = "US"
+
+        self.summary_locations = []
+        self.summary_clinicals = []
+
+    def download_dataset(self, url):
+        r = requests.get(url, allow_redirects=True)
+        tf = tempfile.NamedTemporaryFile(suffix=".gdb.zip", delete=False)
+        with open(tf.name, "wb") as npi_pro_geodatabase:
+            npi_pro_geodatabase.write(r.content)
+
+        return tf.name
+
+    def files_to_submissions(self):
+        print("Getting geodatabase for NPI-PRO dataset...")
+        url = "https://www.arcgis.com/sharing/rest/content/items/7e80baf1773e4fd9b44fe9fb054677db/data"
+        tf = self.download_dataset(url)
+        self.parse_file(file_path=tf)
+
+    def parse_file(self, file_path):
+        # get all the layers from the .gdb file
+        layers = fiona.listlayers(file_path)
```

Comment:
looks like `fiona` is only used for this layer check - i assume the `gdf = gpd.read_file(file_path, layer="NPI202003_TaxonomyGroups")` line would fail anyway if the layer is not there? so maybe we can use exception handling on that line and remove the extra `fiona` dependency?

Reply:
I have checked the doc, it's not required to specify the `layers` at all. 😃